Extensions/TextTruncating.json

```
@@ -0,0 +1,151 @@
+{
+  "author": "",
+  "description": "shorten text content. If text is truncated, it is usually followed with 3 periods called an ellipsis. On webpages, there are several ways to shorten text content so that it fits within a certain designated area.",
+  "extensionNamespace": "",
+  "fullName": "Text Truncate",
+  "helpPath": "",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWZvcm0tdGV4dGJveCIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiPjxwYXRoIGQ9Ik0xNyw3SDIyVjE3SDE3VjE5QTEsMSAwIDAsMCAxOCwyMEgyMFYyMkgxNy41QzE2Ljk1LDIyIDE2LDIxLjU1IDE2LDIxQzE2LDIxLjU1IDE1LjA1LDIyIDE0LjUsMjJIMTJWMjBIMTRBMSwxIDAgMCwwIDE1LDE5VjVBMSwxIDAgMCwwIDE0LDRIMTJWMkgxNC41QzE1LjA1LDIgMTYsMi40NSAxNiwzQzE2LDIuNDUgMTYuOTUsMiAxNy41LDJIMjBWNEgxOEExLDEgMCAwLDAgMTcsNVY3TTIsN0gxM1Y5SDRWMTVIMTNWMTdIMlY3TTIwLDE1VjlIMTdWMTVIMjBaIiAvPjwvc3ZnPg==",
+  "name": "TextTruncating",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/form-textbox.svg",
+  "shortDescription": "shorten text content",
+  "version": "1.0.0",
+  "tags": [
+    "text",
+    "bbcode"
+  ],
+  "authorIds": [
+    "jy7FXnGX0ZZcWfrAI9YuQaeIphi1"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "This shortens a text object (...) ",
+      "fullName": "Truncate Text Object",
+      "functionType": "Action",
+      "name": "Text",
+      "private": false,
+      "sentence": "Truncate _PARAM1_ with a lenght of _PARAM2_followed by_PARAM3_",
```

Careful there are multiple typos and missing spaces.

```suggestion
      "sentence": "Truncate _PARAM1_ to have a maximum length of _PARAM2_ characters followed by _PARAM3_",
```